nengo_viz/swi.py

```diff
@@ -402,14 +408,18 @@ def start(cls, port=80, asynch=True, addr='', browser=False):
             # shut down any remaining threads
             if asynch and server.requests is not None:
                 for _, sock in server.requests:
```

Comment:
It might be fine just to copy the list rather than doing all the lock stuff, but the lock stuff is probably the more proper way to do it.